API/API.ml

```diff
@@ -6,210 +6,257 @@
 (*         *       GNU Lesser General Public License Version 2.1        *)
 (************************************************************************)
 
-module Ppvernac = Ppvernac
-module Command = Command
-module States = States
-module Kindops = Kindops
+(* Warning, this file is ordered in linking order *)
```

Comment:
It might be useful to mention where that linking order is given, so this file can be kept in synch.

Also, at the point where the linking order is given, add a comment that this file needs to change if the order is changed.